python/ray/autoscaler/updater.py

```diff
@@ -273,19 +277,23 @@ def run(self,
                     "SSH command Failed. See above for the output from the"
                     " failure.") from None
 
+    def __get_rsync_ssh_cmd(self, timeout):
+        return " ".join(["ssh"] +
```

Comment:
Try using `shlex.join` here instead. (It handles quoting each argument, so you can avoid extra manual quoting.)

Reply:
According to Python docs, `shlex.join()` is available starting from 3.8 while Ray targets at most 3.6 IIRC.

I thought about using `subprocess.list2cmdline()` originally but then decided to not mix the methods of quoting.

Reply:
Ahh, thanks for noticing that, I completely forgot! Never mind then :)